src/locales/zh_CN/bgm-name.ts

```diff
@@ -0,0 +1,135 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const bgmName: SimpleTranslationEntries = {
+  "music": "Music",
```

Comment:
```suggestion
  "music": "BGM",
```
in zh_cn content we usally use BGM to indicate music here instead of directly translate into "音乐".
BGM has already been a widely used word in Chinese